DTS-STN/canadian-dental-care-plan
# AB#2932 -- Update PowerPlatform country/region mocks

### Description
Updated the country/region MSW, lookup service, and associated pages to conform to Power Platform field names.

### Related Azure Boards Work Items
[AB#2932](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2932)

### Screenshots (if applicable)
N/A

### Checklist
- [x] I have checked that all e2e tests pass by running `npm run test:e2e` (no tests found)

### Test Instructions
Visit address edit page to confirm the dropdowns are functional, and if all unit tests pass.

### Additional Notes
N/A
